stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -559,15 +562,19 @@ public class MessageListViewModel(
                 groupedMessages.add(DateSeparator(message.getCreatedAtOrThrow()))
             }
 
-            groupedMessages.add(
-                MessageItem(
-                    message = message,
-                    groupPosition = position,
-                    parentMessageId = parentMessageId,
-                    isMine = user.id == currentUser?.id,
-                    isInThread = isInThread
+            if (message.isSystem() || message.isError()) {
```

Comment:
Other SDKs including our UI components treat error and system messages in the same way.